### ISSUE TITLE
discussion: Allow each comittee to set admission period outside of the main admission period

### ISSUE THREAD

**Issue by eivinhj** [NTNUI/opptak#127]
Background: For some Comitees the main admission period is not enough. For example does SL (Studentlekene) want a continous admission period after the main admission period, which results in them going back to use Google Forms (https:ntnui.no/sok.sl). 

Where some commitees might have a strict deadline for applicants, others might not (at the same time). 

Questions for discussion: 
1. Should the application system support admissions outside of the main admission period? 
2. If so, should each comittee be able to set their own deadline (or no deadline)?
* If deadlines are different, can they be displayed to the applicants in a good way, or will this create confusion? Maybe the admission system should not show the deadline in this case?
* Maybe the main board can set a common deadline for the main admission periods? Or change the comittees deadlines?

**Comment by eivindkopperud** [comment]
Good point, and I guess it is nice that committees can use the admission system outside of regular admission periods in case they need more people.

An option here is to always show the form as long as there are any committees that accept new applications. In case there is no current admission period, the form could say "The main admission period is over, but some committees still consider applicants"
